build_tools/ci/build_test_cpp.sh

```diff
@@ -7,6 +7,7 @@ repo_root="$(cd $this_dir/../.. && pwd)"
 iree_dir="$(cd $repo_root/../iree && pwd)"
 build_dir="$repo_root/iree-build"
 install_dir="$repo_root/iree-install"
+rm -rf "$build_dir" "$install_dir"
```

Comment:
Is this needed? (just thinking of some poor user who uses it and then deletes everything on what should have been an incremental change)

Reply:
Nah this was just to debug why CI was failing when local was passing. I'll remove